tsconfig.json

```diff
@@ -35,7 +35,7 @@
   "include": ["src/**/*", "src/**/*.json", "packages/**/*", "packages/**/*.json", "scripts/generateAssetData",
     "scripts/generateTradableThorAssetMap"
   ],
-  "exclude": ["dist", "node_modules", "**/__mocks__", "**/*.js"],
+  "exclude": ["dist", "node_modules", "**/__mocks__", "**/*.js", "src/lib/asset-service/service/generatedAssetData.json"],
```

Comment:
Confirmed app still builds and functions fine, and deploys to cloudflare fine on gome.
On the flip side, tsc should be blazingly faster than in develop now vs. borked with this PR.

Expect a follow-up to remove base64 shenanigans - we have none of them in develop as we speak, and this is absolutely not working , see added assets with identicons here

<img width="519" alt="image" src="https://github.com/user-attachments/assets/cd1c14a4-3496-4a60-9368-c58f892dc756">
<img width="967" alt="image" src="https://github.com/user-attachments/assets/a5015f65-7e89-422a-8f83-25b65d9aae0f">